lib/matplotlib/scale.py

```diff
@@ -193,9 +192,9 @@ def set_default_locators_and_formatters(self, axis):
         # update the minor locator for x and y axis based on rcParams
         if (axis.axis_name == 'x' and mpl.rcParams['xtick.minor.visible'] or
                 axis.axis_name == 'y' and mpl.rcParams['ytick.minor.visible']):
-            axis.set_minor_locator(AutoMinorLocator())
+            axis.minorticks_off()
```

Comment:
This should be "on"?  Maybe add a test for this ;-). OTOH This is a change of behaviour, in that it only turns the ticks on, doesn't change then to AutoMinorLocator.  I'm not sure this new behaviour is bad, but it might have ramifications for folks.

Reply:
I think the (existing) test below covers it. That `minorticks_off()` got in through a list minute undo operation. It passes `pytest` on `test_ticker.py`. 

https://github.com/matplotlib/matplotlib/blob/6c3412baf6498d070d76ec60ed399329e6de1b6c/lib/matplotlib/tests/test_ticker.py#L1427-L1442